README.rst

```diff
@@ -100,7 +70,6 @@ Principles
    the process and decide which subjects should be kept for the group level
    analysis.
 
-
 Limitations and reasons not to use *fMRIPrep*
```

Comment:
Can maybe add that it is limited to the availability of non-human templates on templateflow? Of course with a reminder that we encourage others to add more non-human templates!

Reply:
sure, send suggestions! this was just a quick update